resources/base/dialog/ScoreBook_Result.dialog.bxb

```diff
@@ -1,11 +0,0 @@
-dialog (Result) {
```

Comment:
I think as long as the dialog is made of statements that are macros themselves they can still stay in base, but not sure, might be confusing to devs so maybe okay to move them all to target langs.